authors/apps/articles/tests/test_filters.py

```diff
@@ -0,0 +1,71 @@
+from authors.apps.articles.tests.basetests import FilterBaseTest
+from authors.apps.articles.filters import ArticleFilter
+from rest_framework import status
+
+
+class TestFilters(FilterBaseTest):
```

Comment:
Hello @Philipotieno ,
Most 90% of the tests are only for status codes. Would you test data too

Reply:
ok lemme add those too